### ISSUE TITLE
Uncaught ReferenceError: process is not defined (dist/index.es.js & dist/index.js)

### ISSUE THREAD

**Issue by monkeyphysics** [avkonst/hookstate#226]
Webpack 5, custom setup
Chrome: `Uncaught ReferenceError: process is not defined`
Safari: `ReferenceError: Can't find variable: process`

The following code gives an error when `process` is not set, which seems to be the very thing this code tries to protect against 🤷‍♂️

```
var isDevelopmentMode = process !== undefined && process !== null && typeof process === 'object' &&
    process.env !== undefined && process.env !== null && typeof process.env === 'object' &&
    process.env.NODE_ENV === 'development';
```

Obviously there's a `typeof` missing, and it should be:

```
var isDevelopmentMode = typeof process !== undefined && process !== null && typeof process === 'object' &&
    process.env !== undefined && process.env !== null && typeof process.env === 'object' &&
    process.env.NODE_ENV === 'development';
```

**Comment by avkonst** [comment]
Could you please try the latest version. I think it was fixed a while back.

**Comment by avkonst** [comment]
Sorry, the issue is on the latest too. I though this was merged https://github.com/avkonst/hookstate/pull/207 but it turned out it was not.

**Comment by avkonst** [comment]
released in 3.0.11
